README.md

```diff
@@ -28,7 +28,7 @@ dependencies {
             android:value="XXX_api_key_2_XXX"/>
```

Comment:
Is this the api key that returns test ads?

Reply:
yes, but in combination with this user would require correct adUnitId and environment as well for the ad to load, in the documentation that combination is not correct, so, this is just for a demo purpose.